src/main/java/org/sonarlint/intellij/actions/IssuesViewTabOpener.java

```diff
@@ -38,37 +40,62 @@ public IssuesViewTabOpener(Project project) {
   /**
    * Must run in EDT
    */
-  public void open(String tab, boolean switchToChangedFiles) {
+  public void openProjectFiles(String scopeName) {
     ApplicationManager.getApplication().assertIsDispatchThread();
+    ToolWindow toolWindow = getToolWindow();
+    if (toolWindow != null) {
+      toolWindow.show(() -> {
+        JComponent component = selectTab(toolWindow, SonarLintToolWindowFactory.TAB_ANALYSIS_RESULTS);
+        if (component != null) {
+          SonarLintAnalysisResultsPanel panel = (SonarLintAnalysisResultsPanel) component;
+          panel.switchScope(scopeName);
+        }
+      });
+    }
+  }
 
-    ToolWindowManager toolWindowManager = ToolWindowManager.getInstance(project);
-    ToolWindow toolWindow = toolWindowManager.getToolWindow(SonarLintToolWindowFactory.TOOL_WINDOW_ID);
+  /**
+   * Must run in EDT
+   */
+  public void openCurrentFile() {
+    ApplicationManager.getApplication().assertIsDispatchThread();
+    ToolWindow toolWindow = getToolWindow();
     if (toolWindow != null) {
-      toolWindow.show(new ContentSelector(toolWindow, tab, switchToChangedFiles));
+      toolWindow.show(() ->selectTab(toolWindow, SonarLintToolWindowFactory.TAB_CURRENT_FILE));
     }
   }
 
+  private ToolWindow getToolWindow() {
+    ToolWindowManager toolWindowManager = ToolWindowManager.getInstance(project);
+    return toolWindowManager.getToolWindow(SonarLintToolWindowFactory.TOOL_WINDOW_ID);
+  }
+
   private static class ContentSelector implements Runnable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "ContentSelector" class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3985)